AirCasting/Utils/Strings.swift

```diff
@@ -15,23 +15,52 @@ struct Strings {
         static let about: String = "About AirCasting"
     }
     
-    struct BackendSettings {
+    enum BackendSettings {
         static let backendSettings: String = "Backend settings"
         static let Ok: String = "Ok"
         static let Cancel: String = "Cancel"
     }
     
-    struct MyAccountSettings {
+    enum MyAccountSettings {
         static let title: String = "My account"
         static let logStatus: String = "You aren’t currently logged in"
         static let notLogged: String = "You aren’t currently logged in"
         static let createAccount: String = "Create an account"
         static let logIn: String = "Log In"
     }
     
-    struct SignOutSettings {
+    enum SignOutSettings {
         static let title: String = "My account"
         static let Logged: String = "You are currently logged in as admin@admin.com"
         static let signOut: String = "Sign Out"
     }
+    
+    enum ForgotPassword {
+        static let title = "Forgot Password"
+        static let description = "You will get en email with details after 'send new' button pressed"
+        static let actionTitle = "Send new"
+        static let emailInputTitle = "email or username"
+        static let newPasswordSuccessMessage = "Email was sent. Please check your inbox for the details."
+        static let newPasswordSuccessTitle = "Email was sent. Please check your inbox for the details."
```

Comment:
Title is the same as description, is this on purpose?

Reply:
Maybe we could change this text to 
'We've sent you an email with instructions on how to change the password. Please check your inbox.'

Reply:
Unless you want it to stay as it is 😄

Reply:
I made a mistake when changing it to enum, fixed now. Thank You!